GoodreadsEDA/all_bks.py

```diff
@@ -21,52 +21,35 @@ def get_mv_release(bk_m):
                                        WHERE book_title="{bk_m}"
                                        ;""", conn)
         return df_rel["release_date"][0]
-    except IndexError as e:
-        print(f"{e} \n")
+    except IndexError:
+        traceback.print_exc()
         print("""Hmm... it looks as though the program cannot find the
 Release_Date table... ... did you run imdb_release.py yet?""")
 
 
 def create_even_samples(df, df2):
-    """Ballances the data so that the before and after values have an equal #
-    of sample sies."""
+    """Balances the data so that the before and after values have an equal #
+    of sample sites."""
 
     sample_diff = df.shape[0] - df2.shape[0]
     if sample_diff > 0:
-        df = df.sample(df2.shape[0])
-        return (df, df2)
+        return df.sample(df2.shape[0]), df2
     elif sample_diff < 0:
-        df2 = df2.sample(df.shape[0])
-        return (df, df2)
-    else:
-        return (df, df2)
+        return df, df2.sample(df.shape[0])
+    return df, df2
 
 
 # Create empty df to append each book to after preprocessing.
 df_all = pd.DataFrame()
 
 # Go through every title in the list above.
-for i, titl in enumerate(movie_titl):
-    bk = movie_titl[i]
+for title in movie_titles:
     # Get movie release date for movie and df of reviews for the book.
-    conn = sqlite3.connect(f"{os.getcwd()}/review_dbs/reviews.db")
-    mv_release = get_mv_release(bk)
-    # These two cases seemed to be giving me trouble, so I'm just hard coding them
-    # For now.
-    if titl == "Ender’s Game":
```

Comment:
The issue here was that that apostrophe is not ASCII, so it wasn't registering as matching. Same thing in the other title